package.json

```diff
@@ -67,6 +67,7 @@
     "dependencies": {
         "@fontsource/inconsolata": "^4.5.9",
         "@fontsource/inter": "4.5.14",
+        "@prefecthq/graphs": "^0.1.3",
```

Comment:
Just a note. If you have the `^` you gotta makes sure you stick to semver pretty strictly.